.eslintrc

```diff
@@ -0,0 +1,29 @@
+{
+  // tells eslint to use the TypeScript parser
+  "parser": "@typescript-eslint/parser",
+  // tell the TypeScript parser that we want to use JSX syntax
+  "parserOptions": {
+    "tsx": true,
+    "jsx": true,
+    "useJSXTextNode": true,
+    "project": "./tsconfig.json",
+    "tsconfigRootDir": "."
+  },
+  // we want to use the recommended rules provided from the typescript plugin
+  "extends": [
+    "plugin:@typescript-eslint/recommended",
+    "plugin:patternfly-react/recommended"
```

Comment:
Agree, I added it because it was available and I wanted to respect the effort put into those previous rules. No problem removing this or updating those rules to be whatever people think is best. I'll leave in for now and people can feel free to remove it if they don't agree with what the rulesets enforce.

Should we open an issue to update these rules given the latest PF4 efforts? Might be worth discussing this in more detail since it impacts quite a bit of upstream development.